src/main/build/debug.h

```diff
@@ -105,6 +105,7 @@ typedef enum {
     DEBUG_ATTITUDE,
     DEBUG_VTX_MSP,
     DEBUG_GPS_DOP,
+    DEBUG_ANGLEMODE,
```

Comment:
Maybe use non-breaking space as in `DEBUG_ANGLE_MODE`

Reply:
We have `ANGLERATE` so `ANGLEMODE` is more consistent.

Reply:
ANGLEMODE seems to be the most acceptable.

Reply:
Well, no.  Of the 50 or so entries in the list, all follow the convention of using `_` between words, except `CYCLETIME` and `ANGLERATE`, both of which are very old, and are technical debugs that match internal values for which there is no CLI entry where the text is separated by non-breaking spaces .  

All the more recent debug modes separate the words with non-breaking spaces, especially where there is an internal term or a CLI form with non-breaking spaces.  

In this case, the internal form of the term for the mode is `ANGLE_MODE` as in:

```
((FLIGHT_MODE(ANGLE_MODE) && (FLIGHT_MODE(HORIZON_MODE)) && (FLIGHT_MODE(GPS_RESCUE_MODE))
```

Lets keep with the recent convention, and match the existing usage in the code, so that when we go searching for `ANGLE_MODE` we will find all instances.  

Note that a global search for `anglemode` returns nothing in the firmware.

Reply:
@ChrisRosser please make this debug `ANGLE_MODE` with the underscore.